### PR TITLE
doc/ceph orch: Add ceph orch zap command

### PR DESCRIPTION
**Pull request by zdover23**
Add "ceph orch zap" command to the docs.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/mgr/orchestrator.rst [hunk 1]
```diff
@@ -102,6 +102,43 @@ Example::
 .. note::
     Output form Ansible orchestrator
 
+
+
+Erase Devices (Zap Devices)
+---------------------------
+
+Erase (zap) a device so it can be re-used.
```
Comment (sebastian-philipp) [inline doc/mgr/orchestrator.rst:110]: `zap` calls ceph-volume zap on the remote host.

File: doc/mgr/orchestrator.rst [hunk 2]
```diff
@@ -102,6 +102,43 @@ Example::
...
+
+::
+
+     orch device zap
```
Comment (sebastian-philipp) [inline doc/mgr/orchestrator.rst:114]:
```suggestion
     orch device zap <hostname> <path> 
```

File: doc/mgr/orchestrator.rst [hunk 3]
```diff
@@ -102,6 +102,43 @@ Example::
...
+
+Example command::
+
+     ceph orch device zap my_hostname my_path true
```
Comment (sebastian-philipp) [inline doc/mgr/orchestrator.rst:118]:
```suggestion
     ceph orch device zap my_hostname /dev/sda
```

File: doc/mgr/orchestrator.rst [hunk 4]
```diff
@@ -102,6 +102,43 @@ Example::
...
+
+Parameters::
+
+    hostname: (string)
+
+    path: (string)
+
+    force: CephBool
```
Comment (sebastian-philipp) [inline doc/mgr/orchestrator.rst:126]: remove? I don't see the point in adding those trivial parameters.

File: doc/mgr/orchestrator.rst [hunk 5]
```diff
@@ -102,6 +102,43 @@ Example::
...
+Required Permissions::
+
+    rw
+
+Command Flags::
+
+    mgr
+
+
+
```
Comment (sebastian-philipp) [inline doc/mgr/orchestrator.rst:141]: remove. that's part of the autogenerated API documentation

File: doc/mgr/orchestrator.rst (hunk 3 above)
Comment (jschmid1) [reply doc/mgr/orchestrator.rst:118]: we should be careful with `/dev/sda` in the docs. This is usually the root disk. Even though ceph-volume will probably not be able to zap it, we should avoid copy/paste issues.

Suggesting to use `/dev/sdx` or something

**Comment by sebastian-philipp** [comment]
ping @zdover23

**Comment by sebastian-philipp** [comment]
@zdover23 next time, you don't need an extra commit just for the corrections.

**Pull request merged**